### PR TITLE
Add jetbrains mono font ligatures

### PR DESCRIPTION
**Pull request by andreyorst**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -190,6 +190,33 @@ you may find it won't work 100% if you use a different one.
...
+
+This snippet is designed for the *JetBrains Mono* font;
+you may find it won't work 100% if you use a different one.
+
+::
+
+  (use-package ligature
+    :load-path "path-to-ligature-repo"
+    :config
+    ;; Enable all Iosevka ligatures in programming modes
```
Comment (chenl) [inline README.rst:204]: `s/Iosevka/JetBrains mono/`

Comment (andreyorst) [reply README.rst:204]: oops

File: README.rst [hunk 2]
```diff
@@ -190,6 +191,33 @@ you may find it won't work 100% if you use a different one.
...
+This snippet is designed for the *JetBrains Mono* font;
+you may find it won't work 100% if you use a different one.
+
+::
+
+  (use-package ligature
+    :load-path "path-to-ligature-repo"
+    :config
+    ;; Enable all JetBrains Mono ligatures in programming modes
+    (ligature-set-ligatures prog-mode '("-|" "-~" "---" "-<<" "-<" "--" "->" "->>" "-->" "///" "/=" "/=="
```
Comment (mickeynp) [inline README.rst:206]: Nice one. Did you end up enabling the custom `rx` forms I showed you? I think they would serve as a useful example to others. Do you mind adding them?

Comment (andreyorst) [reply README.rst:206]: I did, but I don't use this package for it. It was easier to set composition table manually

**Pull request merged**